Makefile

```diff
@@ -2,33 +2,38 @@ current_user  := $(shell id -u)
 current_group := $(shell id -g)
 BUILD_DIR     := $(PWD)
 DOCKER_FLAGS  := --interactive --tty
-DOCKER_IMAGE  := wikimediade/fundraising-frontend
+DOCKER_IMAGE  := registry.gitlab.com/fun-tech/fundraising-frontend-docker
 
 install-php:
 	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) $(DOCKER_IMAGE):composer composer install $(COMPOSER_FLAGS)
 
 update-php:
 	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) $(DOCKER_IMAGE):composer composer update $(COMPOSER_FLAGS)
 
-ci: covers phpunit cs stan
+ci: phpunit cs stan
 
-test: covers phpunit
+ci-with-coverage: phpunit-with-coverage cs stan
+
+test: phpunit
 
 covers:
-	docker-compose run --rm app ./vendor/bin/covers-validator
+	docker-compose run --rm --no-deps app ./vendor/bin/covers-validator
```

Comment:
if I understood correctly, you removed covers-validator as a dependency,
which is good, because we want to do it according to this ticket https://phabricator.wikimedia.org/T272867,
but all the `covers` tasks should then be removed here too, right?

Reply:
Oops, yes it should